src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,34 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { merge } from '@utilities/merge';
+import { AnimatePresence, motion } from 'framer-motion';
+import React, { ReactElement } from 'react';
+import { ToastProps, toastStylesBackgroundColorsMap } from './types';
+
+export const Toast = ({ isOpen, style, icon, children }: ToastProps): ReactElement => {
+    return (
+        <AnimatePresence>
+            {isOpen && (
+                <motion.div
+                    className="tw-fixed tw-bottom-0 tw-left-[60px] tw-min-h-[4.25rem] tw-m-5 tw-rounded-lg tw-border tw-shadow-lg tw-bg-white tw-border-black-10"
```

Comment:
Positioning of the toast component should be done externally I think. Toast should be able to be placed anywhere. In this case we might need to add an opacity change too 🤔